Country leaders, a few advisors, party names and bugfixes

InfraTheIdiotic opened the pull request:
Additional leaders, characters, party names and some extra minor patches. I have not completed Africa as I felt exhausted and wanted to get this PR out so I can drop the fork and get back to work on Tuva, though please don't take that as pressure as I am not limited for time. The Caribbean is a focus for this contribution.

Let me know if you would rather me add sources.
I do not know any good ways to find errors from my changes in this fork.

**Tuva (largest update):**
- New advisors: Saaya Ala replacing Sodmon Balchor as fortification engineer as well as camo expert general, Naval Preparatory Committee chief of navy requiring a coastline.
- Portraits for advisors
- Added Tyulyush Kechil-ool's unused large portrait
- Removed Kenden Khomushku as a political advisor as the Kamby Lama made him obsolete, remains the Democratic leader
- Separated Lanzy Nazynchap, son of the last Amban-Noyan Sodunam-Balchyr from the Kamby Lama
- Standardised character names: fixed romanisation and removed patronymics, Mongush Bayyskylan's first name and Khertek Anchimaa's name at the time (no -Toka).
- Changed the Bank of Tuva to its official name
- Expanded focus descriptions from my other Tuvan work (still WIP), True Heirs of the Huns is now Appropriate the Gokturks and I added a new national spirit to match it
- Expanded namelist
- Changed the TPRP's name to time-accurate romanisation: Тьва arat-хuviskaalçь nam. The characters don't show everywhere but they work on the tooltip font so I understand if you don't like this.
- Changed non-aligned party to Noyanar (plural of Noyan) - Noyans were Tuvan nobles
- Bug fix: Disorganised Military advisor costs now increase instead of decrease
- Bug fix: State-Funded Projects in the Siberian shared tree no longer shows without NSB
- Bug fix: Fixed easter egg focus localisation
- Bug fix: Assimilate the Altaï is now Assimilate the Altai
- Bug fix: Removed obsolete gender patriarchy locking for Khertek Anchimaa as communist states cannot use that national character, as Khertek is only available for communism

**Greenland:**
- Bad Aksel Svane portrait as the old one was misattributed.

**Malta:**
- Changed Democratic leader to Paul Boffa (Partit Laburista), an earlier PM than Enrico Mizzi whos party lacks the Italian sympathies that got it used as the fascist party. There is a guy interested in making a portrait for him that I'll add in my next contrib unless he finishes in time before this is accepted.
- Moved Enrico Mizzi to the fascist leader to align with the party name, still worked in a democratic system and all WP mentions about him in relation to fascism is accusations so I guess you could imagine he fully commits to fascism if you flip to it?
- Removed redundant DLC lock for the communist leader
- Communist party changed to Maltese (Partit Komunista Malti)
- Generic advisors added.
- Changed Malta's nonaligned name to one reminiscent of Hospitaller Malta's official name with the current name of the SMOM (Monastic State of the Order of Saint John of Jerusalem, of Rhodes and of Malta)
- Added the SMOM as the nonaligned party
- Added the SMOM's Prince and Grand Master, Ludovico Chigi Albani della Rovere, as nonaligned leader (Theocratic Monarchy)
- Changed the Malta Workers Councils to Maltese Workers' Councils as well as the communist name Malta Commune to Maltese Commune, grammatical corrections

**Nenetsia:**
- Added various characters of the 1943 'Mandalada' rebellion. One of its leaders, Sergei Nogo, is the democratic leader with the Freedom Fighter trait and a field marshal. The other leader Vasily Laptander, is also a field marshal and other commanders E. Serotetta, S. Nele and H. Tokpa are generals and military advisors.
- Tyko Vylka is now the communist leader as he served the Soviet government.
- Democratic party is now Maledava, a Nenets term for a peaceful gathering as to opposed to Mandalada, an armed one.
- Generic advisors added separately as some historical characters occupy those positions.

**Jersey-Guernsey:**
- Democratic leader is now Alexander Coutanche, the baliff of Jersey.
- Communist leader is Norman Le Brocq who, though a teenager in 1936 was Jersey's communist party's leader during the occupation. Party name is the semi-fictional Jersey-Guernsey Communist Party.
- Variated the flags and changed the main one to the time-accurate flag of Jersey
-New names:
C: Socialist Federal Republic of Jersey-Guernsey
N: Duchy of Normandy
F: Independent State of Normandy
Neutral and fascist, as they tap into the islands' Norman heritage, could have a decision to core the state of Normandy that I might add with the Tuva update, but if you disagree with giving this tag the name Normandy I can revert it.

**Federated States of Micronesia:**
- Added John Sigrah, the Tokosra (paramount chief) of Kosrae as a staunch monarchist. Comes with an advisor portrait.
- Gave an advisor portrait to Mori Koben.
- Fixed the party names so the vanilla long-form names don't show, and changed Nanpei Regime to Cabinet so it doesn't sound like an authoritarian state.

**Mauritius**:
- Replaced Jules Koenig and the Ralliement Mauricien with Maurice Curé and the Labour Party, founded in 1936 which is considerably earlier than the RM in 1953. The Labour Party was also considerably more prominent. Curé has the Progressive Doctor mp printer trait of Icelandic fame.

**Bahamas:**
- Democratic leader: Henry Milton Taylor of the Progressive Liberal Party (Afro-Bahamian left wing party founded 1953)
- Neutral leader: Roland Theodore Symonette of the United Bahamian Party (White Bahamian right-leaning party founded 1956), not as much of an anachronism as Symonette was head of government in 1955.

**Seychelles:**
- Could not find a leader for the democratic party, the Seychelles Taxpayers and Producers Association, so I went with Marcel Lemarchand, who won a seat for them in the 1948 elections. Editor trait as he was an editor for the newspaper L’Action Catholique. There did exist the League of Coloured Peoples which wasn't for the white minority but if I recall my research a while back correctly it was not specific to Seychelles.

**Palau**:
- Gave the nonaligned party a name - Idid, the ruling clan of Koror.

**Grenada:**
- Democratic leader: Theophilus Albert Marryshow and the Grenada Workingmen's Association, as suggested by Speculoos.

**Saint Vincent and the Grenadines:**
- Dem leader: George Hamilton Charles and his party, the Eighth Army of Liberation, est 1951. Earlier political leader than Ebenezer Joshua.

**Barbados:**
- Leader already in so I added his party, the Barbados Progressive League.

**Saint Lucia:**
- Dem leader: George Charles and the Saint Lucia Labour Party, founded in 1949

**Martinique:**
Both are anachronistic by a couple years unfortunately but both people were active in politics for rt56.
Democratic: Aimé Césaire of the Martinician Progressive Party
Communist: Thélus Léro, who led the Martinician affiliated region of the French Communist party from 1936 to 1946, though the party name is Martinician Communist Party (est. 1957) as it is for an independent state.

**Dominica:**
- Replaced the chief secretary of the Windward Islands Louis Cools-Lartigue with one of the founders of the first party, Phyllis Shand Allfrey of the Dominica Labour Party (founded 1955 with Emmanuel Christopher Loblack though he didn't last). Comes with a unique portrait as there are no female European generic leaders portraits.

**Guadeloupe:**
- Communist leader: Rosan Girard
Anachronistic party (est. 1958), the Guadeloupe Communist Party, done as Girard was an active communist politician in RT56's time under a regional federation of the French Communist Party.

**Antigua and Barbuda:**
- Dem: Vere Bird and the Antigua Labour Party (est. 1946)
- Fixed the adjective and change & to and in the names

**Saint Kitts and Nevis:**
- D: Joseph Matthew Sebastian of the St Kitts Workers' League, suggested by Speculoos.

**Trinidad and Tobago:**
- Democratic: Arthur Andrew Cipriani of the Trinidad Labour Party
- Communist: Replaced Malcolm Nurse who didn't operate in trini with Tubal Uriah Butler who was expelled from the TLP for 'extremism' - he was way more forceful and lead multiple strikes so I made him a syndicalist, see his WP article. Party is the Butler Party, the last name of his own party without allusions to home rule so it makes sense for an independent state. Has a new trait: 'Spiritual Baptist' to represent his role as a preacher of the uniquely West Indian sect or faith. PP factor: +15% and communism drift: +0.05%.
Both come with portraits.

**Jamaica:**
- Leader already in so I added his party, the Jamaica Labour Party.

**Belize:**
- Democratic: John Smith/People's United Party with George Cadle Price, later and more prominent party leader as the democratic reformer
- Non-aligned: W.H. Courtenay/National Party of Belize

**Suriname:**
- Dem: Gerard van der Schroeff and the Nationale Partij Suriname (est. 1946)

**Guyane/Cayenne:**
- Justin Catayée/Parti socialiste guyanais (est.1956), replacing Gaston Monnerville as he was of the Parti radical, which was a French party.
- Renamed the country from Cayenne to Guyane for consistency with the party's name.

**Leeward Federation:**
- Moises Frumencio da Costa Gomez/Nationale Volkspartij (est. 1948). While the National Party was founded earlier in 1944, I went with the NVP to match with the first prime minister, Da Costa Gomez.

**Maldives:**
- Added Muhammad Fareed Didi's party's name: Rayyithunge Muthagaddim Paatee (Democratic)

**Newfoundland:**
- Added United Newfoundland Party, ruling party before loss of self-government

**Aussa:**
- Corrected dynasty name
https://en.wikipedia.org/wiki/Mudaito_dynasty

**Gabon:**
- Léon M'ba's first party, the Gabonese Mixed Committee (though he came to power with the Gabonese Democratic Bloc)

**Congo-Brazzaville:**
- Replaced Jean Malonga, who wasn't a political leader, with Jean Félix-Tchicaya, president of the first party, Parti progressiste congolais.

**Cameroon:**
- Democratic: First prime minister André-Marie Mbida and his party, the Cameroonian Union of Democrats
- Communist: Ruben Um Nyobè, earliest notable gensec of the Union of Peoples of Cameroon
- Removed Seidou Njimoluh Njoya who was the monarch of a small polity, hardly representative of the whole of Cameroon and it conflcited with the nonaligned country name

**Central African Republic:**
- Leader already in, party name: Mouvement pour l'évolution sociale de l'Afrique noire

**Misc:**
- Expanded list of tags with generic advisors in history/general/generic_advisors.txt with countries I have touched that lack them + West Papua, Moluccas, Sunda Kecil, North Borneo and Singapore, as I heeded to a suggestion for Malta to have generic military advisors.
- Mentions of Tunganistan replaced with Altishahr when it refers to the geographical region
- Game rules re-alphabetised, the added where appropriate and names updated, restored Komi and added the GOE countries as they were missing.
- Kashmir and Kashubia's flags have been unswapped.

InfraTheIdiotic committed:
NEN - Nenetsia

Figures of the 1943 Mandalada rebellion. Sergei Nogo and Vasily Laptander were its leaders. Malevada, the new party name, is described in the Mandalada's Russian Wikipedia article as a peaceful gathering of Nenets, while Mandalada is armed.
https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%BD%D0%B4%D0%B0%D0%BB%D0%B0%D0%B4%D0%B0

Fixed the generic advisors so they don't clash with the generals' advisor traits.
Tyko Vylka made into a character and the Communist leader.

InfraTheIdiotic committed:
JER - Jersey-Guernsey

Alexander Coutanche, Bailiff of Jersey as Democratic leader.
Norman Le Brocq, leader of the Jersey Communist Party (resistance movement during Nazi occupation) as Communist leader.

InfraTheIdiotic committed:
Jersey portrait fix

InfraTheIdiotic committed:
FSM - Micronesia

Added portrait for Mori Koben
Added John Sigrah as a staunch monarchist. The Tokosra (king) of Kosrae, which I read was more autocratic and had popularity woes. Comes with a portrait.

InfraTheIdiotic committed:
FSM party names fix

InfraTheIdiotic committed:
Mauritius

Replaced Jules Koenig and the Ralliement Mauricien with Maurice Curé and the Labour Party, founded in 1936 which is considerably earlier than the RM in 1953. The Labour Party was also considerably more prominent.

InfraTheIdiotic committed:
Bahamas

Democratic leader: Henry Milton Taylor of the Progressive Liberal Party (Afro-Bahamian left wing party founded 1953)
Neutral leader: Roland Theodore Symonette of the United Bahamian Party (White Bahamian right-leaning party founded 1956)

InfraTheIdiotic committed:
Seychelles

Could not find a leader for the democratic party, the Seychelles Taxpayers and Producers Association, so I went with Marcel Lemarchand, who won a seat for them in the 1948 elections. There does exist the League of Coloured Peoples but if I recall my research a while back correctly it was not specific to Seychelles.

InfraTheIdiotic committed:
Palau nonaligned part name

Idid clan, the ruling clan of the chiefdom of Koror

InfraTheIdiotic committed:
Grenada

Theophilus Albert Marryshow and the Grenada Workingmen's Association

InfraTheIdiotic committed:
Saint Vincent and the Grenadines

George Hamilton Charles and his party, the Eighth Army of Liberation of 1951 (earlier political leader than Ebenezer Joshua)

InfraTheIdiotic committed:
Barbados

Grantley Herbert Adams' party

InfraTheIdiotic committed:
Saint Lucia

George Charles and the Saint Lucia Labour Party, founded in 1949

InfraTheIdiotic committed:
Martinique

Both are anachronistic by a couple years unfortunately
Democratic: Aimé Césaire of the Martinician Progressive Party
Communist: Thélus Léro, who led the Martinician affiliated region of the French Communist party from 1936 to 1946, though the party name is Martinician Communist Party (est. 1957) as it is for an independent state.

InfraTheIdiotic committed:
Dominica

Replaced the chief secretary of the Windward Islands Louis Cools-Lartigue with one of the founders of the first party, Phyllis Shand Allfrey of the Dominica Labour Party (founded 1955 with Emmanuel Christopher Loblack though he didn't last). Due to the lack of female generic leader portraits I used one of the Kaiserreich operative ones as a placeholder until I figure out what to do or make one of probably awful quality.

InfraTheIdiotic committed:
Guadeloupe

Communist leader: Rosan Girard
Anachronistic party (est. 1958), the Guadeloupe Communist Party, done as Girard was an active communist politician in RT56's time under a regional federation of the French Communist Party.

InfraTheIdiotic committed:
Antigua and Saint Kitts

Vere Bird and the Antigua Labour Party (est. 1946)

Joseph Matthew Sebastian of the St Kitts Workers' League

InfraTheIdiotic committed:
Dominica gender fix

InfraTheIdiotic committed:
Trinidad and Tobago

Democratic: Arthur Andrew Cipriani of the Trinidad Labour Party
Communist: Replaced Malcolm Nurse who didn't operate in trini with Tubal Uriah Butler who was expelled from the TLP for 'extremism' - he was way more forceful and lead multiple strikes so I made him a syndicalist, see his WP article. Party is the Butler Party, the last name of his own party without allusions to home rule so it makes sense for an independent state.

InfraTheIdiotic committed:
Jamaica

Leader already in, just the Jamaica Labour Party

InfraTheIdiotic committed:
Trait for Tubal Butler

InfraTheIdiotic committed:
Trini portraits and trait fix

InfraTheIdiotic committed:
Awful Axel Svane portrait to replace the misattributed one

InfraTheIdiotic committed:
Belize

Democratic: John Smith/People's United Party with George Cadle Price, later and more prominent party leader as the democratic reformer
Non-aligned: W.H. Courtenay/National Party of Belize

InfraTheIdiotic committed:
Suriname

Gerard van der Schroeff/Nationale Partij Suriname (est. 1946)

InfraTheIdiotic committed:
French Guiana (Guyane/Cayenne)

Justin Catayée/Parti socialiste guyanais (est.1956), replacing Gaston Monnerville as he was of the Parti radical, which was a French party.

Renamed the country from Cayenne to Guyane for consistency with the party's name.

InfraTheIdiotic committed:
Netherlands Antilles

Moises Frumencio da Costa Gomez/Nationale Volkspartij (est. 1948)
While the National Party was founded earlier in 1944, I went with the NVP to match with the first prime minister, Da Costa Gomez.

InfraTheIdiotic committed:
Maldivian democratic party

InfraTheIdiotic committed:
Newfoundland party

United Newfoundland Party, ruling party before loss of self-government

InfraTheIdiotic committed:
Aussa

Correct dynasty name
https://en.wikipedia.org/wiki/Mudaito_dynasty

InfraTheIdiotic committed:
Gabon

Léon M'ba's first party, the Gabonese Mixed Committee (though he came to power with the Gabonese Democratic Bloc)

InfraTheIdiotic committed:
Congo-Brazzaville

Replaced Jean Malonga, who wasn't a political leader, with Jean Félix-Tchicaya, president of the first party, Parti progressiste congolais.

InfraTheIdiotic committed:
Cameroon

First prime minister André-Marie Mbida and his party, the Cameroonian Union of Democrats
Communist: Ruben Um Nyobè, earliest notable gensec of the Union of Peoples of Cameroon
Removed Seidou Njimoluh Njoya who was the monarch of a small polity, hardly representative of the whole of Cameroon and it conflcited with the nonaligned country name

InfraTheIdiotic committed:
CAR democratic party

Leader already in, party name: Mouvement pour l'évolution sociale de l'Afrique noire

InfraTheIdiotic committed:
TUVA (LARGE UPDATE)

NOTE: To fix some of the localisation conflicts in advisor names, I have added the ideas_l_english.yml file from Vanilla to the mod as Waffle suggested, let me know if you want that changed.

New advisors: Saaya Ala replacing Sodmon Balchor as fortification engineer as well as camo expert general, Naval Preparatory Committee chief of navy requiring a coastline.
Portraits for advisors
Added Tyulyush Kechil-ool's unused large portrait
Removed Kenden Khomushku as a political advisor as the Kamby Lama made him obsolete, remains the Democratic leader
Separated Lanzy Nazynchap, son of the last Amban-Noyan Sodunam-Balchyr from the Kamby Lama
Standardised character names: fixed romanisation and removed patronymics, Mongush Bayyskylan's first name and Khertek Anchimaa's name at the time (no -Toka).
Changed the Bank of Tuva to its official name
Expanded focus descriptions from my other Tuvan work (still WIP), True Heirs of the Huns is now Appropriate the Gokturks and I added a new national spirit to match it
Expanded namelist
Changed the TPRP's name to time-accurate romanisation: Тьва arat-хuviskaalçь nam. The characters don't show everywhere but they work on the tooltip font
Changed non-aligned party to Noyanar (plural of Noyan) - Noyans were Tuvan nobles
Bug fix: Disorganised Military advisor costs now increase instead of decrease
Bug fix: State-Funded Projects in the Siberian shared tree no longer shows without NSB
Bug fix: Fixed easter egg focus localisation
Bug fix: Assimilate the Altaï  is now Assimilate the Altai
Bug fix: Removed obsolete gender patriarchy locking for Khertek Anchimaa as communist states cannot use that national character, as Khertek is only available for communism

InfraTheIdiotic committed:
Phyllis Shand Allfrey portrait (DMN)

InfraTheIdiotic committed:
Reverted Cameroon ideology change for consistency

InfraTheIdiotic committed:
Removing Tunganistan mentions and polishing country boost game rules

Tunganistan replaced with Altishahr when it refers to the geographical region
Game rules re-alphabetised, the added where appropriate and names updated

InfraTheIdiotic committed:
Malta

Democratic leader changed to Paul Boffa, earlier PM than Enrico Mizzi and not sympathetic to Italy. Parties reorganised and changed to Maltese language

InfraTheIdiotic committed:
Antigua name fix

InfraTheIdiotic committed:
More generic advisors

InfraTheIdiotic committed:
Update MLT - Malta.txt

Moved Enrico Mizzi to the fascist leader to align with the party name, still worked in a democratic system and all WP mentions about him in relation to fascism is accusations so I guess you could imagine he fully commits to fascism if you flip to it?

InfraTheIdiotic committed:
Khalistan difficulty option

InfraTheIdiotic committed:
Malta portrait fix

InfraTheIdiotic committed:
Kashmir/Kashubia flag fix

InfraTheIdiotic committed:
Kashmir difficulty slider

InfraTheIdiotic committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

marijn211 reviewed (commented):
Error log seems clear but please avoid using the obsolete country leader system

InfraTheIdiotic commented:
That can be changed, will fix that and let you know once it's done

Commit:
Merge branch 'Beta-Edits' into pr/3176

BiccyThiccy committed:
Merge branch 'Beta-Edits' into pr/3176

InfraTheIdiotic committed:
JER characters

InfraTheIdiotic committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

InfraTheIdiotic committed:
BAH characters

InfraTheIdiotic committed:
MAU characters

InfraTheIdiotic committed:
SEY characters

InfraTheIdiotic committed:
Merge branch 'Beta-Edits' of https://github.com/InfraTheIdiotic/1956_infrish_characters_05_2025 into Beta-Edits

InfraTheIdiotic committed:
Merge branch 'Beta-Edits' into Beta-Edits

InfraTheIdiotic committed:
BRB, GRA, SVG characters

InfraTheIdiotic committed:
Barbados fix

InfraTheIdiotic committed:
STL characters

InfraTheIdiotic committed:
Extra: Jersey-Guernsey new flags and names

Variated the flags
New names:
C: Socialist Federal Republic of Jersey-Guernsey
N: Duchy of Normandy
F: Independent State of Normandy
Neutral and fascist, as they tap into the islands' Norman heritage, could have a decision to core the state of Normandy that I might add with the Tuva update, but if you disagree with giving this tag the name Normandy I can revert it.

InfraTheIdiotic commented:
Update: Modified Jersey-Guernsey, see its section in the description for the changes

Commit:
Merge branch 'Beta-Edits' into pr/3176

InfraTheIdiotic committed:
STL fix

InfraTheIdiotic commented:
I have undone a La Resistance DLC-lock for Malta's communist leader (Malta Workers Councils) as it seemed redundant to me, let me know if that's there for a purpose.

InfraTheIdiotic commented:
Also adding the Prince and Grand Master of the Sovereign Military Order of Malta Ludovico Chigi Albani della Rovere as the non-aligned leader

InfraTheIdiotic committed:
Loads of characters!!!

InfraTheIdiotic committed:
Malta characters and extra additions

Changed Malta's nonaligned name to one reminiscent of Hospitaller Malta's official name with the current name of the SMOM (Monastic State of the Order of Saint John of Jerusalem, of Rhodes and of Malta)
Added the SMOM as the nonaligned party
Added the SMOM's Prince and Grand Master, Ludovico Chigi Albani della Rovere, as nonaligned leader (Theocratic Monarchy)
Changed the Malta Workers Councils to Maltese Workers' Councils as well as the communist name Malta Commune to Maltese Commune, grammatical corrections

InfraTheIdiotic committed:
Merge branch 'Beta-Edits' of https://github.com/InfraTheIdiotic/1956_infrish_characters_05_2025 into Beta-Edits

InfraTheIdiotic committed:
Rest of the GOE states' difficulty settings because I forgot they were in the mod

InfraTheIdiotic commented:
How do I change it back to open if I am able to?!/1/?!/1!?

InfraTheIdiotic commented:
nevermind that

Commit:
Merge branch 'Beta-Edits' into pr/3176

InfraTheIdiotic committed:
Merge branch 'Beta-Edits' into Beta-Edits

Commit:
Merge branch 'Beta-Edits' into pr/3176

Commit:
Fixed bad advise from Waffle

Commit:
Merge branch 'Beta-Edits' into pr/3176

Commit:
Readded new stuff to decrease pr difference

InfraTheIdiotic commented:
Nenetsia was intentionally removed from the generic leaders as I added some historical replacements to them but not for all, so I removed it to stop duplicates and manually re-added the generics as characters. 

I edited this to say sorry for not pointing it out on the patch notes but I just realised I did but i guess i could have made that clearer.

InfraTheIdiotic commented:
Is there a way that could have been done better? I will change it back if not.

Xaelorichero-02 commented:
> Nenetsia was intentionally removed from the generic leaders as I added some historical replacements to them but not for all, so I removed it to stop duplicates and manually re-added the generics as characters.
> 
> I edited this to say sorry for not pointing it out on the patch notes but I just realised I did but i guess i could have made that clearer.

Just remove it. I though you added it in. It was my mistake when resolving conflicts.

InfraTheIdiotic committed:
Removed NEN from generic advisors again

InfraTheIdiotic commented:
Done

marijn211 committed:
Merge branch 'Beta-Edits' into pr/3176

marijn211 reviewed (approved):
I think I did a diff check previously
Branch is up to date
Will check error log
In-game check shouldnt be necessary

marijn211 reviewed (changes requested):
[20:55:59][1936.01.01.12][effectimplementation.cpp:5413]: history/countries/GDL - Guadeloupe.txt:5: recruit_character: Unknown character 
[20:55:59][1936.01.01.12][effectimplementation.cpp:5413]: history/countries/GDL - Guadeloupe.txt:6: recruit_character: Unknown character 

(Theres another but thats just a duplicate recruitment error)

InfraTheIdiotic commented:
On it

InfraTheIdiotic committed:
Removed replaced characters

InfraTheIdiotic committed:
Merge branch 'Beta-Edits' of https://github.com/InfraTheIdiotic/1956_infrish_characters_05_2025 into Beta-Edits

InfraTheIdiotic commented:
Done
